src/components/header/index.js

```diff
@@ -12,7 +12,9 @@ import { useRef, useState } from "react"
 const Header = () => {
     const { token, onLogout } = useAuth()
     const [isMenuVisible, setIsMenuVisible] = useState(false)
-    const menuRef =useRef(null)
+    const menuRef = useRef(null)
+    const { loggedInUserInfo } = useAuth()
+    const user = loggedInUserInfo
```

Comment:
This can be simplified :-) 

```suggestion
    const { loggedInUserInfo: user } = useAuth()
```

This extracts the value of `loggedInUserInfo` from the `useAuth()` object, and then assigns it to a variable called `user`.

Reply:
added.